packages/ui/src/views/repo/webhooks/webhook-create/repo-webhook-create-page.tsx

```diff
@@ -96,15 +99,15 @@ export const RepoWebhooksCreatePage: FC<RepoWebhooksCreatePageProps> = ({
           ? t('views:repos.editWebhookTitle', 'Webhook details')
           : t('views:repos.createWebhookTitle', 'Create a webhook')}
       </h1>
-      <FormWrapper onSubmit={handleSubmit(onSubmit)}>
+      <FormWrapper {...formMethods} onSubmit={handleSubmit(onSubmit)}>
         <Fieldset>
           <WebhookToggleField register={register} setValue={setValue} watch={watch} t={t} />
         </Fieldset>
         <Fieldset>
-          <WebhookNameField register={register} errors={errors} disabled={false} t={t} />
+          <WebhookNameField register={register} disabled={false} t={t} />
```

Comment:
This implies that `disabled` is the default state 🤔 
```suggestion
          <WebhookNameField register={register} t={t} />
```

Reply:
For some reason `disabled` prop was kept as `required` in types by the developer. Removed it now.